src/main/java/org/sagebionetworks/bridge/models/apps/Exporter3Configuration.java

```diff
@@ -1,20 +1,21 @@
 package org.sagebionetworks.bridge.models.apps;
 
-import java.util.Objects;
-
 /** This class holds Exporter 3.0 configuration for a given app. */
 public final class Exporter3Configuration {
     private String createStudyNotificationTopicArn;
     private Long dataAccessTeamId;
     private String participantVersionTableId;
+    private String participantVersionDemographicsTableId;
+    private String participantVersionDemographicsViewId;
     private String projectId;
     private String rawDataFolderId;
     private Long storageLocationId;
 
     /** Helper method that returns true if all configuration attributes are specified. */
     public boolean isConfigured() {
-        return dataAccessTeamId != null && participantVersionTableId != null && projectId != null &&
-                rawDataFolderId != null && storageLocationId != null;
+        return dataAccessTeamId != null && participantVersionTableId != null
+                && participantVersionDemographicsTableId != null && participantVersionDemographicsViewId != null
+                && projectId != null && rawDataFolderId != null && storageLocationId != null;
```

Comment:
Ah I see the issue. What if we re-run `initExporter3` from the worker in the case where Exporter3 is enabled but `Exporter3Configuration` is not configured? I think `initExporter3` was designed with this case in mind:

```
     * (b) If in the future, we add something new (like a notification queue, or a default view), we can re-run this
     * API to create the new stuff without affecting the old stuff.
```

That would also avoid the case where existing studies using Exporter3 expect their demographics to be exported, but they would not be because those Synapse resources would never be created, so the check to see if they exist would always be false, and therefore demographics wouldn't be exported.

Reply:
I'm not sure that's a good idea. As the Exporter scales up, there will be more requests coming in at the same time, for the same app, for possibly overlapping studies. Plus, there's the weird circular dependency between the Server, the Exporter, and the Server again.

It would be simpler to just not export Demographics unless it's been configured.